include/zephyr/bluetooth/hci.h

```diff
@@ -1766,9 +1766,10 @@ struct bt_hci_rp_le_per_adv_set_info_transfer {
 	uint16_t conn_handle;
 } __packed;
 
-#define BT_HCI_LE_PAST_MODE_NO_SYNC              0x00
-#define BT_HCI_LE_PAST_MODE_NO_REPORTS           0x01
-#define BT_HCI_LE_PAST_MODE_SYNC                 0x02
+#define BT_HCI_LE_PAST_MODE_NO_SYNC                 0x00
+#define BT_HCI_LE_PAST_MODE_SYNC_NO_REPORTS         0x01
+#define BT_HCI_LE_PAST_MODE_SYNC_REPORTS_UNFILTERED 0x02
+#define BT_HCI_LE_PAST_MODE_SYNC_REPORTS_FILTERED   0x03
```

Comment:
This is modifying values in a public API file. 

I would suggest to simply add `BT_HCI_LE_PAST_MODE_SYNC_FILTERED` instead of modifying `BT_HCI_LE_PAST_MODE_SYNC` as well.

Reply:
I've removed the changes, now only adding the new value (and formatting the old ones)